tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -87,7 +87,7 @@ Given('user {string} from remote server has shared {string} with user {string} f
  * @param {string} elementToShare  path of file/folder being shared
  * @param {string} sharer  username of the sharer
  * @param receiver  username of the receiver
- * @param shareType  Type of share 0 = user, 1 = group, 3 = public (link), 6 = federated (cloud share).
+ * @param {number} shareType  type of share user, group, public (link), federated (cloud share).
```

Comment:
why changing that comment? how is this change relevant to this PR?

Reply:
My bad, I thought this would be revealing information
reverted back to as it is!